src/api/layers/ExternalGroupOfLayers.class.js

```diff
@@ -0,0 +1,23 @@
+import ExternalLayer from '@/api/layers/ExternalLayer.class'
+import LayerTypes from '@/api/layers/LayerTypes.enum'
+
+/**
+ * Description of a group of layers, that could be added altogether or separately, that stems from a getCapabilities XML parsing. (see https://www.mediamaps.ch/ogc/schemas-xsdoc/sld/1.2/capabilities_1_3_0_xsd.html#Layer)
+ *
+ * If the group of layer is added to the map, all layers being part of it should be added under this group's name "banner"
+ */
+export default class ExternalGroupOfLayers extends ExternalLayer {
+    /**
+     * @param {String} name Name of this layer to be shown to the user
+     * @param {String} hostname getCapabilities URL host name, so that it can be used in the ID generation
+     * @param {ExternalLayer[]} layers Description of the layers being part of this group (they will all be displayed at the same time, in contrast to an aggregate layer)
+     */
+    constructor(name, hostname, ...layers) {
+        super(name, LayerTypes.GROUP, `${hostname}:${name.replaceAll(' ', '_')}`, null, 1, true)
+        this.layers = [...layers]
+    }
+
+    getID() {
+        return this.externalLayerId
+    }
+}
```

Comment:
For which use case is this ExternalGroupOfLayers ? From a OOP stand point it is wrong to inherit from `ExternalLayer` here.

Reply:
It's for the external layer UI tool, as some Kanton stores layers as group (we do not, AFAIK)
But you are correct that this could easily be renamed `GroupOfLayers` and extend from `AbstractLayer` (also using this kind of type for the internal layers). I might do that it's clearer this way.

Reply:
As discussed ok for now